clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -176,12 +171,11 @@ export class CopybookDownloadService implements vscode.Disposable {
       copybookprofile.profile,
       dataset,
       copybookprofile.getCopybook(),
+      this.storagePath,
     );
     if (!fs.existsSync(copybookPath)) {
```

Comment:
Make sure that the directory exists.

Reply:
which directory ?

Reply:
The one where `copybookPath` should be in.

Reply:
it is handled here : https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/pull/2265/files/e19f57f598c4397cf6f2c88f497cab5bdb423cf1#diff-c9b74f01a4b43df586e9ec106f05a4c816e4e617eb66a7ea0cff49b0f009f18dL228-L234

api will create if it is missing. we only need to provide proper download options including correct path.